bloom-housing/bloom
# fix: app due date and time fixes

## Description

I noticed this morning that our application due date checks do not take the due time into account. This fixes that. I originally opened this off of master, so had to rebase off of dev. As a result, for some reason files like the listing e2e tests are showing changes, but they're already on there.

## How Can This Be Tested/Reviewed?

Create or update a listing with a due date and time and you should see the due date checks using the time. The listing on public should correctly reflect the status based off of the due time.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
